supply-chain/audits.toml

```diff
@@ -762,6 +762,21 @@ is similar to what it once was back then. Skimming over the crate there is
 nothing suspicious and it's everything you'd expect a Rust URL parser to be.
 """
 
+[[audits.wasm-coredump-builder]]
+who = "Sven Sauleau <sven@cloudflare.com>"
+criteria = "safe-to-deploy"
+version = "0.1.10"
+
+[[audits.wasm-coredump-encoder]]
+who = "Sven Sauleau <sven@cloudflare.com>"
+criteria = "safe-to-deploy"
+version = "0.1.10"
+
+[[audits.wasm-coredump-types]]
+who = "Sven Sauleau <sven@cloudflare.com>"
+criteria = "safe-to-deploy"
+version = "0.1.10"
+
```

Comment:
For now we're enforcing that the edits to the `supply-chain/*` folder are only done by "trusted reviewers" of the project since this affects our supply-chain security (and others who import these audits). To that end thanks for adding these here, but our current process (which isn't well documented) is that if a contributor needs new vet entries we [post a separate PR](https://github.com/bytecodealliance/wasmtime/pull/5878) which the contributor's PR is then rebased on top of.

Once that PR lands, though, you should be able to rebase this PR on top of that one and back out the changes to the `supply-chain/*` directory.

Reply:
Ok, thanks.